src/django_linear_migrations/apps.py

```diff
@@ -121,6 +134,34 @@ def check_max_migration_files(
     else:
         app_config_set = set()
 
+    migration_loader = MigrationLoader(
+        connections[DEFAULT_DB_ALIAS], ignore_no_migrations=True
+    )
+    app_names = [a.label for a in first_party_app_configs()]
+    conflicts = migration_loader.detect_conflicts()
+    if app_names:
+        conflicts = {
+            app_label: conflict
+            for app_label, conflict in conflicts.items()
+            if app_label in app_names
+        }
+    if conflicts:
+        conflict_msg = "; ".join(
+            f"{', '.join(names)} in {app}" for app, names in conflicts.items()
+        )
+        errors.append(
+            Error(
+                id="dlm.E005",
+                msg=(
+                    "Conflicting migrations detected;"
+                    + f" multiple leaf nodes in the migration graph: {conflict_msg}."
+                ),
+                hint="Run 'python manage.py makemigrations --merge'",
```

Comment:
This isn't always the way to solve conflicts, I think we can at least phrase as "Fix with e.g. './manage.py makemigrations --merge'."